project/main.gd

```diff
@@ -24,51 +24,51 @@ func _ready() -> void:
 
 func _init_level_choice_popup() -> void:
 	var popup: PopupMenu = level_choice.get_popup()
-	popup.add_item("DEBUG", Sentry.LEVEL_DEBUG + 1)
-	popup.add_item("INFO", Sentry.LEVEL_INFO + 1)
-	popup.add_item("WARNING", Sentry.LEVEL_WARNING + 1)
-	popup.add_item("ERROR", Sentry.LEVEL_ERROR + 1)
-	popup.add_item("FATAL", Sentry.LEVEL_FATAL + 1)
+	popup.add_item("DEBUG", SentrySDK.LEVEL_DEBUG + 1)
+	popup.add_item("INFO", SentrySDK.LEVEL_INFO + 1)
+	popup.add_item("WARNING", SentrySDK.LEVEL_WARNING + 1)
+	popup.add_item("ERROR", SentrySDK.LEVEL_ERROR + 1)
+	popup.add_item("FATAL", SentrySDK.LEVEL_FATAL + 1)
 
-	_on_level_choice_id_pressed(Sentry.LEVEL_INFO + 1)
+	_on_level_choice_id_pressed(SentrySDK.LEVEL_INFO + 1)
 
 
 func _update_user_info() -> void:
 	# The user info is persisted in the user data directory (referenced by "user://"),
 	# so it will be loaded again on subsequent launches.
-	var user: SentryUser = Sentry.get_user()
+	var user: SentryUser = SentrySDK.get_user()
 	username.text = user.username
 	email.text = user.email
 	user_id.text = user.id
 
 
 func _on_level_choice_id_pressed(id: int) -> void:
-	_event_level = (id - 1) as Sentry.Level
+	_event_level = (id - 1) as SentrySDK.Level
 	match _event_level:
-		Sentry.LEVEL_DEBUG:
+		SentrySDK.LEVEL_DEBUG:
 			level_choice.text = "DEBUG"
-		Sentry.LEVEL_INFO:
+		SentrySDK.LEVEL_INFO:
 			level_choice.text = "INFO"
-		Sentry.LEVEL_WARNING:
+		SentrySDK.LEVEL_WARNING:
 			level_choice.text = "WARNING"
-		Sentry.LEVEL_ERROR:
+		SentrySDK.LEVEL_ERROR:
 			level_choice.text = "ERROR"
-		Sentry.LEVEL_FATAL:
+		SentrySDK.LEVEL_FATAL:
 			level_choice.text = "FATAL"
 
 
 func _on_capture_button_pressed() -> void:
-	Sentry.capture_message(message_edit.text, _event_level)
-	print("Captured message event. Event ID: " + Sentry.get_last_event_id())
+	SentrySDK.capture_message(message_edit.text, _event_level)
+	print("Captured message event. Event ID: " + SentrySDK.get_last_event_id())
```

Comment:
I addressed it in a follow-up commit. AFAIK we don't have a Guid in Godot ecosystem. We currently use `String` to represent ID. Another option is to use a custom class - although it may be an overkill.

Reply:
It is sourced by `sentry-native` for `NativeSDK` backend, so it should be alright.